zilliqa/src/api/zil.rs

```diff
@@ -894,3 +896,34 @@ fn get_num_ds_blocks(_params: Params, node: &Arc<Mutex<Node>>) -> Result<String>
     let num_ds_blocks = (num_tx_blocks / TX_BLOCKS_PER_DS_BLOCK) + 1;
     Ok(num_ds_blocks.to_string())
 }
+
+fn get_recent_transactions(
+    _params: Params,
+    node: &Arc<Mutex<Node>>,
+) -> Result<RecentTransactionsResponse> {
+    let node = node.lock().unwrap();
+    let mut block_number = node.get_chain_tip();
+    let mut txns = Vec::new();
+    while block_number > 0 && txns.len() < 100 {
```

Comment:
Can we add a limiter for the number of blocks too? I worry about a network with many empty blocks (like the current proto-testnet) in which this loop will take a long time to terminate. 100 blocks seems reasonable to me, with the justification that transactions aren't really 'recent' if they occurred ages ago?

Reply:
Yep sounds good will do

Reply:
Done. The docs are also updated to match